apiserver/resources_test.go

```diff
@@ -135,11 +136,12 @@ func (s *resourceUploadSuite) makeUploadArgs(c *gc.C) url.Values {
 	q.Add("revision", "3")
 	q.Add("size", fmt.Sprint(len(content)))
 	q.Add("fingerprint", fp.Hex())
+	q.Add("timestamp", fmt.Sprint(time.Now().UnixNano()))
```

Comment:
I know you are comparing for empty, so it's ok for now... But maybe the clock will help in the future?..